### DISCUSSION TITLE
Hypercore compatibility

### DISCUSSION THREAD

**Discussion by samuk** [markqvist/Reticulum#480] [Ideas]
It looks like reticulum would be [compatible](https://github.com/holepunchto/hypercore/issues/338) with [Hypercore](https://docs.pears.com/#building-blocks)

So if some hypercore nodes existed on a Reticulum network you could use your Reticulum identity to write to the Hypercore append only log.

I'm not sure exactly what that buys you, but there might be some applications where that's interesting?

**Comment by markqvist** [comment]
Hey, that's a really interesting observation, and I think it holds technically as well :) I'm not sure exactly how their implementation works, but from the looks of it, it should be totally possible to use Reticulum identities there 1:1.

No idea either, but I'm sure there's someone who can come up with interesting uses for this! Thanks for sharing it!
